refactor: removed unused dependencies

Nagarjuna0033 opened the pull request:
Fixes - [Jira-#129](https://mifosforge.jira.com/browse/MM-129)

Didn't create a Jira ticket, click [here](https://mifosforge.jira.com/jira/software/c/projects/MM/issues/) to create new.

Please Add Screenshots If there are any UI changes.

| Before                                     | After                                  |
|--------------------------------------------|----------------------------------------|
|                                            |                                        |


Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Run the static analysis check `./gradlew check` or `ci-prepush.sh` to make sure you didn't break anything

- [ ] If you have multiple commits please combine them into one commit by squashing them.

Nagarjuna0033 committed:
refactor: removed unused dependencies

Nagarjuna0033 commented:
@niyajali , I have removed some unused dependencies Can you please look into this.

niyajali commented:
@Nagarjuna0033 Fix workflows as we renamed the repository and haven't updated in this project on kmp-branch,

 for reference take a look into dev branch

Nagarjuna0033 commented:
> @Nagarjuna0033 Fix workflows as we renamed the repository and haven't updated in this project on kmp-branch,
> 
> for reference take a look into dev branch

you mean to do in this PR

Nagarjuna0033 committed:
fix: pr check workflow

Nagarjuna0033 commented:
> @Nagarjuna0033 Fix workflows as we renamed the repository and haven't updated in this project on kmp-branch,
> 
> for reference take a look into dev branch

@niyajali I have fixed have a look once
